plugin/pkg/admission/limitranger/admission.go

```diff
@@ -548,39 +543,15 @@ func PodValidateLimitFunc(limitRange *corev1.LimitRange, pod *api.Pod) error {
 	return utilerrors.NewAggregate(errs)
 }
 
-type podResourcesOptions struct {
-	// InPlacePodVerticalScalingEnabled indicates that the in-place pod vertical scaling feature gate is enabled.
-	InPlacePodVerticalScalingEnabled bool
-}
-
 // podRequests is a simplified version of pkg/api/v1/resource/PodRequests that operates against the core version of
 // pod. Any changes to that calculation should be reflected here.
```

Comment:
the resources helpers now has an option `UseStatusResources`, instead of just checking the feature gate. I think this function is actually equivalent to calling the common one with `UseStatusResoruces: false`. I'll still make a note of it though.

Reply:
SG. I think having the feature gates in the option could be misleading. Once those features graduate to GA, folks may simply assume that they are always true without thinking twice.